spec/system/casa_cases/edit_spec.rb

```diff
@@ -303,6 +303,33 @@ def sign_in_and_assign_volunteer
         expect(page).to have_text(volunteer_2.display_name)
       end
     end
+
+    context "deleting court mandates", js: true do
+      let(:casa_case) { create(:casa_case) }
+      let(:mandate) { build(:case_court_mandate) }
+
+      before do
+        casa_case.case_court_mandates << mandate
+      end
```

Comment:
This logic is repeated in your tests. It'd be nice to avoid this repetition by using a trait within your factory.